### ISSUE TITLE
Packet drop in Sx1302

### ISSUE THREAD

**Issue by vikasrohal** [Lora-net/sx1302_hal#23]
After multiple testing I have observed that there are some LoRa packets drops in gateway designed from sx1302 , in comparison with sx1301 for which I have also attached some screenshots. There are min of 5% packets drops in sx1302 . I hope that someone have faced similar issue can help me in some way. thanks for help in advance. 

![Screenshot (38)](https://user-images.githubusercontent.com/23522407/96590506-b9533600-1303-11eb-97bc-aaeccddcc446.png)
![Screenshot (43)](https://user-images.githubusercontent.com/23522407/96590509-bb1cf980-1303-11eb-92c7-95e805f7c2b8.png)

**Comment by rony-vargas** [comment]
I have experience similar issue, I have a RAK2245 and RAK2287 (SX1301 vs SX1301) and the first captures way more packets than the second. I am looking various forks from this repo to see if there is any improvement that I can implement locally

**Comment by rony-vargas** [comment]
Hi @vikasrohal , what packet forwarder are you using? could you print the output when the concentrator SX1302 starts? I am interested to see the `rssi_offset` in my case I suspect it could be part of the problem, I am using the `basicstation` as packet forwarder and it is not honoring the `rssi_offset` of -215.4 set in the `station.conf`, configurations may if you're using a different paket forwarder, but in some place of a you should define a rssi_offset based on your board's desing.
Since I believe it is more related the specific forwarder I am using I have opened an issue in that repository:
[https://github.com/lorabasics/basicstation/issues/80](url)

**Comment by rony-vargas** [comment]
@vikasrohal I found my issue to be in the node side, it was using a wrong preamble in the code, LoRaWAN establishes that it needs to be 8 Symbols for preamble and in my case the node had a code bug that reported more than that. It made the Gateway to discard that message. After I fixed the Preamble length in the node's code, I had no more packet drop.
Apparently the `rssi_offset` has nothing to do with this issue, but still looking forward the TheThingsStack to avoid pushing this for the LNS protocol and avoid hardware issue.
